lib/bitstyles_phoenix/components/form.ex

```diff
@@ -56,6 +56,20 @@ defmodule BitstylesPhoenix.Form do
       ~s(<label for="user_name">Name</label><input id="user_name" maxlength="255" name="user[name]" type="text">)
   """)
 
+  story("Text field with error", """
+      iex> safe_to_string ui_input(@error_form, :name)
+      ~s(<label for="user_name">Name</label><input id="user_name" maxlength="255" name="user[name]" type="text"><span class="u-fg--warning" phx-feedback-for="user_name">is too short</span>)
+  """)
+
+  story("Text field with multiple errors", """
+      iex> safe_to_string ui_input(@error_form, :email)
+      ~s(<label for="user_email">Email</label><input id="user_email" maxlength="255" name="user[email]" type="text"><ul>
+        <li><span class=\"u-fg--warning\" phx-feedback-for=\"user_email\">is invalid</span></li>
+        <li><span class=\"u-fg--warning\" phx-feedback-for=\"user_email\">must end with @bitcrowd.net</span></li>
```

Comment:
```suggestion
        <li><span class="u-fg--warning" phx-feedback-for="user_email">is invalid</span></li>
        <li><span class="u-fg--warning" phx-feedback-for="user_email">must end with @bitcrowd.net</span></li>
```

no? why do these need to be escaped?

Reply:
I also had to escape the classname I added to the `ul`, so it seems legit 🤔

Reply:
🤔  But the whole thing is in a `~s()` sigil and the first line of it is not escaped?!

I mean, this is tested, right? So it has to be correct :)

Reply:
huh, yes true there’s no escaping on the label & input 🤔 But if the escaping is not required, normally the tests explode complaining about the `\\\"` (presumably from multiple escaping)

…I’ll try it out locally again later today — if I can remove these escaping, I will